Sources/VersionCheck/Helpers/StringHelper.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  File.swift
+//  
+//
+//  Created by Emad Ghorbania on 16/11/2022.
+//
+
+import Foundation
+extension String {
+    func versionCompare(_ otherVersion: String) -> ComparisonResult {
```

Comment:
What if `otherVersion` is empty? Should the function guard against it and give a "domain error" about it?

Reply:
I don't know, it's sdk function that I use, I assume it says something! but I won't provide this function to outside so there is no need to worry about that or add a test or something!

Reply:
You could also check if the input version string has the correct x.x.x structure and throw an error if not. Just an idea, not needed at all

Reply:
No you have the opportunity to have the version in x.x or x as well that's why I have that extension